build-scripts/createRelease.ts

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as child_process from 'child_process'
+import * as fs from 'fs-extra'
+import * as path from 'path'
+
+const changesDirectory = '.changes'
+const nextReleaseDirectory = path.join(changesDirectory, 'next-release')
+// tslint:disable-next-line:no-var-requires no-unsafe-any
+const releaseVersion = require(path.join('..', 'package.json')).version
+const changesFile = path.join(changesDirectory, `${releaseVersion}.json`)
+
+fs.mkdirpSync(nextReleaseDirectory)
+
+const changeFiles = fs.readdirSync(nextReleaseDirectory)
+if (changeFiles.length === 0) {
+    console.log('Error! no changes to release!')
+    process.exit(-1)
+}
+try {
+    fs.accessSync(changesFile)
+    console.log(`Error! changelog file ${changesFile} already exists for version ${releaseVersion}!`)
+    process.exit(-1)
+} catch (err) {
+    // This is what we want to happen, the file should not exist
+}
+
+const now = new Date()
+const timestamp = `${now.getFullYear()}-${now.getMonth()}-${now.getDay()}`
```

Comment:
1. getDay returns 0-6 day of week
1. change this to produce yyyy-mm-dd (2019-07-01) instead of yyyy-m-d (2019-7-1)
1. ensure this isn't emitting 0-indexed values

Reply:
fixed